spec/javascripts/tests/Header_spec.js

```diff
@@ -0,0 +1,65 @@
+describe('Header', function() {
+  'use strict';
+
+  beforeEach(function(done) {
+    var self = this;
+
+    fixture.setBase('spec/javascripts/fixtures');
+
+    requirejs(
+      ['jquery', 'Header'],
+      function($, Header) {
+        fixture.load('Header.html');
+
+        self.$header = $(fixture.el).find('[data-dough-component="Header"]');
+        self.obj = new Header(self.$header);
+
+
+        self.activeClass = 'mobile-header__search-active';
+        self.$header = $(document).find('[data-site-header]');
+        self.$mobileSearchButton = self.$header.find('[data-mobile-search-button]');
+
+        var $openText = $(fixture.el).find('[data-mobile-search-text]');
+        self.expected = {
+          openText: "Open Search",
+          closeText: "Close Search",
+          openTextFromDiv: $openText.html(),
+          closeTextFromDiv: $openText.attr("data-mobile-search-text-close")
+        }
+
+        done();
+      }, done);
+  });
+
+  afterEach(function() {
+    fixture.cleanup();
+  });
+
+  describe('Search Bar Initialise', function() {
+    it('Checks the open search text is correct', function() {
+      this.obj.init();
```

Comment:
For efficiency the init() method can be added to the beforeEach() method: self.obj.init();

Reply:
config change b87229d breaks tests, removed all tests 60ba64d